src/post-processor/postProcessorManager.ts

```diff
@@ -3,11 +3,13 @@ import { MetadataRepository } from '../metadata/MetadataRepository'
 import type { Work } from '../types/work'
 
 import BaseProcessor from './baseProcessor'
+import DeactivateFlowProcessor from './deactivateFlowProcessor'
 import FlowTranslationProcessor from './flowTranslationProcessor'
 import IncludeProcessor from './includeProcessor'
 import PackageGenerator from './packageGenerator'
 
 const processors: Array<typeof BaseProcessor> = [
+  DeactivateFlowProcessor,
```

Comment:
**question:** I wonder if it should be done after the FlowTranslation 🤔 ?

In this case we don't care about translations right ?